plugins/inputs/phpfpm/phpfpm_test.go

```diff
@@ -358,3 +359,14 @@ max active processes: 1
 max children reached: 2
 slow requests:        1
 `
+
+func TestPhpFpmLogWithoutPanic(t *testing.T) {
+	p := &phpfpm{}
+	// AddInput sets the Logger
+	if err := shim.New().AddInput(p); err != nil {
+		t.Error(err)
+		return
+	}
+
+	p.Log.Debug("test")
+}
```

Comment:
I'd prefer we drop this test, as this is not something we have in every other PR. And instead put up a test case that fails to decode the JSON, which would hit the log error.

Reply:
I changed the test as suggested.